components/eamxx/src/dynamics/homme/eamxx_homme_process_interface.hpp

```diff
@@ -89,6 +90,11 @@ class HommeDynamics : public AtmosphereProcess
   void rayleigh_friction_init ();
   void rayleigh_friction_apply (const Real dt) const;
 
+  // Set IOP object if necessary
+  void set_intensive_observational_period (const std::shared_ptr<control::IntensiveObservationPeriod>& iop) {
```

Comment:
This could be done just in the base class, no? You will end up setting it for all atm procs, but who cares? It will reduce code duplication (even more if a new parametrization had to support iop)